src/components/FooterContainer/styles.js

```diff
@@ -0,0 +1,126 @@
+export const footer = {
+  display: "flex",
+  justifyContent: "center",
+  backgroundColor: "#F5FBFC",
+  color: "#333",
+  width: "100%",
+  flexDirection: "column",
+  alignItems: "center",
+};
+
+export const footerContainer = {
+  display: "flex",
+  width: 1200,
+  height: 232,
+  justifyContent: "space-between",
+};
+
+export const footerLeft = {
+  width: 584,
+};
+
+export const footerRight = {
+  display: "flex",
+  justifyContent: "space-around",
+};
+
+export const logo = {
+  justifyContent: "center",
+  alignItems: "center",
+};
+export const logoIcon = {
+  width: 40,
+  height: 51,
+};
+
+export const footerColumn = {
+  width: 190,
+};
+
+export const footerColumnList = {
+  listStyle: "none",
+  padding: 0,
+  margin: 0,
+  display: "flex",
+  justifyContent: "space-between",
+  flexDirection: "column",
+};
+
+export const footerLink = {
+  color: "#333",
+  textDecoration: "none",
+};
+
+export const footerColumnName = {
+  color: "#08090A",
+  fontFamily: "Inter",
+  fontSize: 13,
+  fontStyle: "normal",
+  fontWeight: 400,
+  letterSpacing: 1.5,
+  textTransform: "uppercase",
+  margin: 0,
+};
+
+export const footerColumnListLi = {
+  marginTop: 28,
+
+  color: "#546285",
+  fontFamily: "Inter",
+  fontSize: 15,
+  fontStyle: "normal",
+  fontWeight: 400,
+};
+
+export const footerIcon = {
+  width: 18,
+  height: 18,
+};
+
+export const footerSign = {
+  display: "flex",
+  justifyContent: "space-between",
+  flexDirection: "row",
+  alignItems: "flex-end",
+  width: 1200,
+  height: 101,
+};
+
+export const footerIconLink = {
+  display: "flex",
+  justifyContent: "center",
+  alignItems: "center",
+  width: 40,
+  height: 40,
+
+  borderRadius: 100,
+  border: "1px solid rgba(126, 130, 143, 0.30)",
+};
+
+export const footerIconLinkContainer = {
+  display: "flex",
+  justifyContent: "center",
+  alignItems: "center",
+  width: 168,
+  listStyle: "none",
+  padding: 0,
+  margin: 0,
+};
+
+export const pFooterSign = {
+  color: "#35B8BE",
+  textAlign: "center",
+  fontFamily: "Inter",
+  fontSize: 16,
+  fontStyle: "normal",
+  fontWeight: 400,
+  letterSpacing: 0.36,
+  margin: 0,
+  padding: 0,
+  marginLeft: 10,
+  marginRight: 10,
+};
+
+export const spanFooterSign = {
+  display: "flex",
+};
```

Comment:
I don't really like this approach, because it is a inline styles, which difficult to beat due to highest specificity